ligo/README.md

```diff
@@ -79,6 +79,11 @@ These configurable options are:
 - the `contract_name` is the alias that the originated contract will be associated with.
 - the `destination` is the directory where the steps files will be saved.
 
+## Generating Typescript interface types
```

Comment:
As far as "ligo" is concerned this may be sufficient, however we do need a `README` in the `typescript` directory as well.

It should mention:
1. a short description
2. dependencies: `yarn`, `node`, etc.
3. how to build/install (the info in the PR description seems to be outdated BTW)
4. a list of its features ("recognition" and entrypoints) and an example on how to use each (one entrypoint call is enough)

In short some basic documentation around this SDK

Reply:
Done.